pinot-core/src/main/java/org/apache/pinot/core/operator/InstanceResponseOperator.java

```diff
@@ -93,4 +108,16 @@ public static long calTotalThreadCpuTimeNs(long totalWallClockTimeNs, long multi
   public String getOperatorName() {
     return OPERATOR_NAME;
   }
+
+  private void prefetchAll() {
+    for (int i = 0; i < _fetchContexts.size(); i++) {
```

Comment:
(nit) put `_fetchContexts.size()` in a local variable

Reply:
fixed